inputs/values/resources_hg38.json

```diff
@@ -23,7 +22,6 @@
   "mei_bed" : "gs://gcp-public-data--broad-references/hg38/v0/sv-resources/resources/v1/mei_hg38.bed.gz",
   "melt_std_vcf_header" : "gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/melt_standard_vcf_header.txt",
   "noncoding_bed" : "gs://gcp-public-data--broad-references/hg38/v0/sv-resources/resources/v1/noncoding.sort.hg38.bed",
-  "pesr_exclude_list" : "gs://gatk-sv-resources-public/hg38/v0/sv-resources/resources/v1/PESR.encode.peri_all.repeats.delly.hg38.blacklist.sorted.bed.gz",
```

Comment:
I know the file name has Delly in it, but this is used in several places unrelated to Delly, so we need to keep it. (And make sure to re-build the inputs and run the validate script)

Reply:
@epiercehoffman , thanks for your review! I applied your comment in [this commit ](749aec92478cc198195e8ece013c630635f85525)